compiler/rustc_codegen_llvm/src/gotoc/metadata.rs

```diff
@@ -179,7 +101,7 @@ impl<'tcx> GotocCtx<'tcx> {
     pub fn fname_option(&self) -> Option<String> {
         // The function name is contained in the context member named instance,
         // and instance is defined only for function contexts.
-        self.current_fn.as_ref().map(|x| self.symbol_name(x.instance))
+        self.current_fn.as_ref().map(|x| self.symbol_name(x.instance()))
```

Comment:
Is this not the same as `self.current_fn.name`?

Reply:
`self.current_fn.map(|x| x.name())`

Reply:
Done. This function is only used in one place, so I just moved the usage to there.